docs/source/markdown/options/health-cmd.md

```diff
@@ -10,3 +10,5 @@ to be applied. A value of **none** disables existing healthchecks.
 
 Multiple options can be passed in the form of a JSON array; otherwise, the command is interpreted
 as an argument to **/bin/sh -c**.
+
+Note: The Healthcheck does not inherit the configuration form image.
```

Comment:
Is this only if the `--health-cmd` option is passed? I'm a little confused as to what this means

Reply:
The default values are used when using the `--helath-cmd`  flag. The default values are used even if Healthcheck is configured in the image. 

With other flags, you can change the configuration of the Healthcheck that is baked in the image. For example, just override the interval of Healthcheck. 

I will reword the change in the document.